src/main/java/net/sf/jabref/importer/fetcher/OAI2Fetcher.java

```diff
@@ -207,7 +207,7 @@ public BibEntry importOai2Entry(String key) {
             InputStream inputStream = oai2Connection.getInputStream();
 
             /* create an empty BibEntry and set the oai2identifier field */
-            BibEntry be = new BibEntry(IdGenerator.next(), BibtexEntryTypes.ARTICLE);
+            BibEntry be = new BibEntry(IdGenerator.next(), BibtexEntryTypes.ARTICLE.getName());
```

Comment:
There are quite a few places with "getName" instead of the string literal (in the test for example).